extensions/wikia/ImageReview/ImageReviewHelperBase.class.php

```diff
@@ -1,64 +0,0 @@
-<?php
-
-abstract class ImageReviewHelperBase extends WikiaModel {

```

Comment:
Again, the code here has been moved into different classes. Also, the inheritance here just doesn't make a lot of sense in the way it's being used.